server/src/main/java/org/opensearch/wlm/WorkloadManagementSettings.java

```diff
@@ -22,16 +23,66 @@ public class WorkloadManagementSettings {
     private static final Double DEFAULT_NODE_LEVEL_MEMORY_CANCELLATION_THRESHOLD = 0.9;
     private static final Double DEFAULT_NODE_LEVEL_CPU_REJECTION_THRESHOLD = 0.8;
     private static final Double DEFAULT_NODE_LEVEL_CPU_CANCELLATION_THRESHOLD = 0.9;
+    private static final Long DEFAULT_QUERYGROUP_SERVICE_RUN_INTERVAL_MILLIS = 1000L;
     public static final double NODE_LEVEL_MEMORY_CANCELLATION_THRESHOLD_MAX_VALUE = 0.95;
     public static final double NODE_LEVEL_MEMORY_REJECTION_THRESHOLD_MAX_VALUE = 0.9;
     public static final double NODE_LEVEL_CPU_CANCELLATION_THRESHOLD_MAX_VALUE = 0.95;
     public static final double NODE_LEVEL_CPU_REJECTION_THRESHOLD_MAX_VALUE = 0.9;
+    public static final String DEFAULT_WLM_MODE = "monitor_only";
 
     private Double nodeLevelMemoryCancellationThreshold;
     private Double nodeLevelMemoryRejectionThreshold;
     private Double nodeLevelCpuCancellationThreshold;
     private Double nodeLevelCpuRejectionThreshold;
 
+    /**
+     * Setting name for QueryGroupService node duress streak
+     */
+    public static final String QUERYGROUP_DURESS_STREAK_SETTING_NAME = "wlm.query_group.duress_streak";
+    private int duressStreak;
+    public static final Setting<Integer> QUERYGROUP_SERVICE_DURESS_STREAK_SETTING = Setting.intSetting(
+        QUERYGROUP_DURESS_STREAK_SETTING_NAME,
+        3,
+        3,
+        Setting.Property.Dynamic,
+        Setting.Property.NodeScope
+    );
+
+    /**
+     * Setting name for Query Group Service run interval
+     */
+    public static final String QUERYGROUP_ENFORCEMENT_INTERVAL_SETTING_NAME = "wlm.query_group.enforcement_interval";
```

Comment:
Just thinking again on this `enforcement_interval` might be confused with the resiliency mode, since this interval is independent of monitor_only, soft, enforced mode. Does `evaluation_interval` look better in that sense?

Reply:
resilency modes are SOFT and HARD hence I think it wouldn't confuse the enforcement interval with mode.